### PR TITLE
updating examples.rst

### PR DESCRIPTION
**Pull request by jposada202020**
Hello, I am going to submit one more PR adding an example, just wanted to correct this first. Tks :)

### CODE REVIEW

File: docs/examples.rst [hunk 1]
```diff
@@ -6,3 +6,21 @@ Ensure your device works with this simple test.
...
+Disklogger
+------------
+
+Logging data to .CSV file on CircuitPython Disk
+
+.. literalinclude:: ../examples/csv_disklogger.py
+    :caption: examples/csv_disklogger.py
+    :linenos:
+
+Dictwriter test
```
Comment (tekktrik) [inline docs/examples.rst]: Would you mind fixing this to `DictWriter` to match the name?  Thanks!

**Review by tekktrik — CHANGES_REQUESTED** [review]
One minor fix - thanks for this!

File: docs/examples.rst (hunk 1 above)
Comment (jposada202020) [reply docs/examples.rst]: Ok thanks. see https://github.com/tekktrik/CircuitPython_CSV/pull/18/commits/69a254e732be329b57c9364be094c3552ee00c3b

**Review by tekktrik — APPROVED** [review]
Thanks for the improved documentation!

**Pull request merged**